fe/fe-core/src/main/java/com/starrocks/connector/iceberg/ScalarOperatorToIcebergExpr.java

```diff
@@ -226,6 +227,10 @@ public Expression visitInPredicate(InPredicateOperator operator, IcebergContext
                         return literalValue;
                     }).collect(Collectors.toList());
 
+            if (literalValues.stream().anyMatch(Objects::isNull)) {
+                return null;
+            }
```

Comment:
Some functions within `ScalarOperatorFunctions` could be computed on frontends. We can then obtain the computed result and transform it into an Iceberg expression next.

Reply:
Agree.
We can skip it first. I added TODO, then we will continue optimizing according to actual needs.